README.md

```diff
@@ -6,7 +6,7 @@ Tools to prepare Presto releases.
 
 To download the release tools executable.
 ```
-curl -L -o /tmp/presto_release "https://oss.sonatype.org/service/local/artifact/maven/redirect?g=com.facebook.presto&a=presto-release-tools&v=LATEST&r=snapshots&c=executable&e=jar"
+curl -L -o /tmp/presto_release "https://oss.sonatype.org/service/local/artifact/maven/redirect?g=com.facebook.presto&a=presto-release-tools&v=RELEASE&r=releases&c=executable&e=jar"
```

Comment:
Is this redundant with the script? Should it just point to the script and people can edit it?

Reply:
The README file has instructions to run commands other than generating release notes, so I think we should include it. If people wants to cut it release, it doesn't make much sense to ask them to take a look of the `release-notes` shell script for the download instruction, and then follow `README` to run the command line tool.